### DISCUSSION TITLE
Which specific tags are not considered by API Untagged

### DISCUSSION THREAD

**Discussion by fatecyx** [Difegue/LANraragi#897] [Q&A] [answered]
When I was using the Batch page, I noticed that “Archives with no tags have been pre-checked”, but some of mangas have already been set tags artist/group/series. Which specific tags are not considered?

**Comment by Difegue** [comment, marked as answer]
See matching code in Stats:  https://github.com/Difegue/LANraragi/blob/d2ab6807cc4b1ed1fe902c264cba7750ae07f435/lib/LANraragi/Model/Stats.pm#L77
